common/core/hwclayer.cpp

```diff
@@ -211,24 +211,11 @@ void HwcLayer::SufaceDamageTransfrom() {
   HwcRect<int> translated_damage =
       TranslateRect(surface_damage_, -source_crop_.left, -source_crop_.top);
 
-  int display_width = display_frame_.right - display_frame_.left;
-  int display_height = display_frame_.bottom - display_frame_.top;
-  int source_width = source_crop_.right - source_crop_.left;
-  int source_height = source_crop_.bottom - source_crop_.top;
-
   // From observation: In Android, when the source crop doesn't
```

Comment:
The proposed code change does not seem to match the comment.  If I understand the change correctly, if the surface_damage_ is populated with data, and the source_crop coordinate is (0,0), no translation is needed.  Therefore, should the preceding comment be changed to the following?

  // From observation: In Android, when the source crop coordinate
  // is (0, 0), the surface damage is already translated to global display
  // coordinate.  Therefore, no translation is needed.

Reply:
Hi @MicheleL-Intel, you are right, i will modify the comment according. 
Besides, the rotation situation is not verified so far as the rotation is not supported on P. Just leave the rotation situation here temporary. 
Thanks for your advise.